package.json

```diff
@@ -4,32 +4,34 @@
   "description": "An Application Architecture Using Backbone.js",
   "repository": "git://github.com/chaplinjs/chaplin.git",
   "devDependencies": {
+    "backbone": "~1.2.3",
+    "backbone.nativeview": "~0.3.3",
+    "chai": "3.5.x",
+    "coffee-coverage": "1.0.1",
+    "coffee-script": "1.10.x",
+    "coffeeify": "~2.0.1",
     "grunt": "~0.4.5",
+    "grunt-browserify": "~4.0.1",
     "grunt-cli": "~0.1.13",
     "grunt-coffeelint": "~0.0.15",
-    "grunt-mocha-test": "~0.12.7",
-    "grunt-browserify": "~4.0.1",
-    "grunt-transbrute": "0.2.x",
     "grunt-contrib-compress": "0.14.x",
-    "grunt-contrib-watch": "~0.6.1",
     "grunt-contrib-uglify": "~0.11.1",
-    "coffeeify": "~2.0.1",
+    "grunt-contrib-watch": "~0.6.1",
+    "grunt-istanbul": "0.7.0",
+    "grunt-mocha-test": "~0.12.7",
+    "grunt-transbrute": "0.2.x",
+    "jquery": "2.2.x",
     "jsdom": "~8.0.2",
     "jsdom-assign": "shvaikalesh/jsdom-assign",
-    "coffee-script": "1.10.x",
     "mocha": "~2.4.5",
+    "prompt": "~0.2.14",
     "sinon": "~1.17.3",
     "sinon-chai": "2.8.x",
-    "jquery": "2.2.x",
-    "chai": "3.5.x",
-    "backbone": "~1.2.3",
-    "backbone.nativeview": "~0.3.3",
-    "underscore": "~1.8.3",
-    "prompt": "~0.2.14"
+    "underscore": "~1.8.3"
   },
   "main": "build/chaplin.js",
   "scripts": {
-    "test": "grunt test && grunt test:jquery",
+    "test": "grunt test && grunt test:jquery && grunt coverage",
```

Comment:
Thought it was convenient to have the coverage in the same task that run both native and jQuery test, but I agree this not totally necessary to run in the CI.

This could be useful if we want to plug on coverall or something similar though.